src/openforms/config/templatetags/privacy_policy.py

```diff
@@ -0,0 +1,18 @@
+from django import template
+from django.template import Context, Template
+
+from ..models import GlobalConfiguration
+
+register = template.Library()
+
+
+@register.simple_tag()
+def privacybeleid():
```

Comment:
I think I prefer the English name as with most (all?) template tags we use now.

So, `privacy_policy`